### PR TITLE
Update README file SN-140

### PR DESCRIPTION
**Pull request by danyducky**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,16 +33,33 @@ Installation
...
 ```
 
-Commands
+Library upload preparation
+--------
+
+* Create `nuspec` template file. Add it into library folder.
+> See an example file [here](https://github.com/Saritasa/SaritasaTools/blob/master/src/Saritasa.Tools.Common/Saritasa.Tools.Common.nuspec.template).
+
+* Create `Changelog` & `Version` files. Add it into library folder.
```
Comment (krasninja) [inline README.md]: Upper case (CHANGELOG, VERSION).

File: README.md [hunk 2]
```diff
@@ -33,16 +33,33 @@ Installation
 PM> Install-Package Saritasa.Tools.<PackageName>
 ```
 
-Commands
+Library upload preparation
+--------
```
Comment (krasninja) [inline README.md]: Pls add the same number of `-` as header above.

File: README.md [hunk 3]
```diff
@@ -33,16 +33,33 @@ Installation
 PM> Install-Package Saritasa.Tools.<PackageName>
 ```
 
-Commands
+Library upload preparation
```
Comment (krasninja) [inline README.md]: [EBI] Package Preparation

File: README.md [hunk 4]
```diff
@@ -33,16 +33,33 @@ Installation
...
+--------
+
+* Create `nuspec` template file. Add it into library folder.
```
Comment (krasninja) [inline README.md]: Since these are steps, use 1, 2, 3 syntax.

**Review by ko-vasilev — APPROVED** [review]

**Pull request merged**